fe/projects/AspNetCore/src/areas/Orders/components/NewOrderForm/NewOrderForm.js

```diff
@@ -420,6 +436,17 @@ function NewOrderForm(props) {
                                         setMoreInfo(e.target.value);
                                     }} />
                             </div>
+                            <div className="column is-2 is-flex is-align-items-flex-end">
```

Comment:
**suggestion:** Checkbox should be displayed only when selected product is from stock